suplemon/main.py

```diff
@@ -530,14 +533,12 @@ def trigger_event_after(self, event):
     def toggle_fullscreen(self):
         """Toggle full screen editor."""
         display = self.config["display"]
+        show_indicators = True
         if display["show_top_bar"]:
-            display["show_top_bar"] = 0
-            display["show_bottom_bar"] = 0
-            display["show_legend"] = 0
-        else:
-            display["show_top_bar"] = 1
-            display["show_bottom_bar"] = 1
-            display["show_legend"] = 1
+            show_indicators = False
```

Comment:
This can be compacted even further:

**Ternary:**

```python
show_indicators = False if display["show_top_bar"] else True
```

**Boolean:**

```python
show_indicators = not display["show_top_bar"]
```

Reply:
Nice! I like the python ternary syntax, but I think the `not` style is a lot better for this case. I would go with that.

This PR has really helped me think about how things should be done (in the best and cleanest possible way). The impact it's made on me is probably even bigger than the impact to this project. Thanks @Gnewbee and @twolfson :slightly_smiling_face: